.github/workflows/docker-publish.yml

```diff
@@ -18,11 +18,24 @@ env:
     GIT_COMMITTER_EMAIL: 'apps@dhis2.org'
 
 jobs:
-    publish:
+    release:
```

Comment:
I split these on two, since failing to deploy to Aws shouldn't prevent tagging and releasing to GitHub.